### DISCUSSION TITLE
save space/full screen apps between different monitor setups

### DISCUSSION THREAD

**Discussion by glennverhalle** [rxhanson/Rectangle#357] [Ideas]
I have 3 setups I use on a regular base with my MBPro
- Office : 2 external monitors
- home office: 1 external monitor
- on the road: no external monitors

I have a picky setup on what I want in terms of spaces (FS = Full Screen)

Monitor 1 (biggest) FS app1 - FS app2 - Desktop 1 - FS app3 - FS app 4 - Desktop 2
Monitor 2 (smallest) Desktop 3 - FS app 5 - Desktop 4
Laptop : FS app 6 - Desktop 5

when only 1 external monitor I move the spaces from monitor 2 to the laptop

Each time I connect the laptop at the office or home-office I have to re-arrange all spaces to have them in the correct order and on the correct monitor. Is this something you could add to the app ? Even if I would need to choose a menu or click a button to save/restore my windows/spaces that would be already great.

**Comment by rxhanson** [comment]
Unfortunately, I don't have plans to add any support for Spaces at this time. 

Here's an issue that I'm leaving open in regard to Spaces:
https://github.com/rxhanson/Rectangle/issues/215

If anything changes, I'll update that issue.
